### PR TITLE
Bump alpine to 3.17.2

### PR DESCRIPTION
**Pull request by ivantenevvasilev**
**Description**

Changes proposed in this pull request:
- bump alpine from `3.17.1` to `3.17.2`

**Related issue(s)**
- #2868

**Pull Request status**

- [x] [N/A] Implementation
- [x] [N/A] Unit tests
- [x] [N/A] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [x] [N/A] Mocks are regenerated, using the automated script

### CODE REVIEW

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -159,11 +159,11 @@ global:
...
-      version: "PR-2886"
+      version: "PR-2897"
       name: compass-schema-migrator
     system_broker:
       dir:
-      version: "PR-2876"
+      version: "PR-2897"
       name: compass-system-broker
     certs_setup_job:
       containerRegistry:
```
Comment (dzahariev) [inline chart/compass/values.yaml:169]: Bump also external services mock version.

**Review by dzahariev — CHANGES_REQUESTED** [review]

**Review by dzahariev — APPROVED** [review]

**Review by nyordanoff — DISMISSED** [review]

**Review by dzahariev — APPROVED** [review]

**Review by StanislavStefanov — APPROVED** [review]

**Pull request merged**